packages/api/src/index.js

```diff
@@ -76,7 +79,10 @@ r.add(
 const psa = (handler, mode) =>
   withPsaErrorHandler(withPinningAuthorized(withMode(handler, mode)))
 
+r.add('get', '/did', withMode(did, RO), [postCors])
```

Comment:
can we use `didGet` for consistency with other handlers namings?

Reply:
all the handlers follow the file name

Reply:
yes, we can also rename the filename.

Reply:
what im saying is that the current file names and function name are consistent with the other endpoint